featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1831,30 +1841,17 @@ def test_calc_feature_matrix_with_cutoff_df_and_instance_ids(es):
     assert (feature_matrix[property_feature.get_name()] == labels).values.all()
 
 
-def test_calculate_feature_matrix_returns_default_values():
-    transactions = pd.DataFrame({
-        "id": [1, 2, 3, 4],
-        "session_id": ["a", "a", "b", "c"],
-        "value": [1, 1, 1, 1]
-    })
-
-    sessions = pd.DataFrame({
-        "id": ["a", "b"]
-    })
-
-    es = ft.EntitySet()
-    es.entity_from_dataframe(entity_id="transactions",
-                             dataframe=transactions,
-                             index="id")
-    es.entity_from_dataframe(entity_id="sessions",
-                             dataframe=sessions,
-                             index="id")
+def test_calculate_feature_matrix_returns_default_values(pd_transactions_defaultvalue_es):
```

Comment:
the fixture should be `transactions_defaultvalue_es` so that all 3 versions get tested, not just pandas

Reply:
The dask dataframes looses index information when we convert from pandas to dask. I found this is a common problem outside as well: https://stackoverflow.com/questions/60070170/losing-index-information-when-using-dask-dataframe-to-parquet-with-partitionin

Do we have a way to resolve this internally?

Reply:
@seriallazer Are you referring to the `to_pandas(feature_matrix)` call on line 1853? If so here is some info that might help. For Dask and Koalas inputs, Featuretools does not set the index of the returned feature matrix like it does for pandas. Also row order is not guaranteed with Dask and Koalas. The `to_pandas` helper function has a couple optional parameters that can help in this case. Since you are setting the index values to be `id` when creating the entityset, you can update your call to this instead to set the index and make sure it is sorted in the original order:
```python
feature_matrix = to_pandas(feature_matrix, index='id, sort_index=True)
```